resources/building-mobile-extensions/get-the-application-context.md

```diff
@@ -0,0 +1,36 @@
+# Get the Application Context
+
+In Android, you can retrieve the `android.app.Application` instance using the `MobileCore.getApplication()` API. The returned Application represents the base class which maintains the global application state. For more information, [Moble Core API Reference](../../using-mobile-extensions/mobile-core/configuration-reference/mobile-core-api-reference.md)
```

Comment:
- In the 2nd sentence, does "Application" have to be capitalized? if not, please change to a lower case "a". 
- In the 3rd sentence, please add "see" after the comma.